types/block.go

```diff
@@ -68,3 +74,166 @@ type Signature []byte
 type IntermediateStateRoots struct {
 	RawRootsList [][]byte
 }
+
+func (h *Header) ToProto() *pb.Header {
+	return &pb.Header{
+		Version: &pb.Version{
+			Block: h.Version.Block,
+			App:   h.Version.App,
+		},
+		NamespaceId:     h.NamespaceID[:],
```

Comment:
IIRC, the convention is golang is `Id` instead of `ID`.

Reply:
IIRC I uppercased this because of linter crying out loud ;)
Also see: https://github.com/golang/lint/issues/124